app/bundles/CampaignBundle/Tests/Executioner/EventExecutionerTest.php

```diff
@@ -40,59 +40,26 @@
 
 class EventExecutionerTest extends \PHPUnit\Framework\TestCase
 {
-    /**
```

Comment:
@kuzmany not sure if it was intentional to remove these lines or if it was the same issue as with #8809